sllurp/llrp_proto.py

```diff
@@ -1909,6 +1972,29 @@ def encode_AISpec(par, param_info):
     return encode_all_parameters(par, param_info, b''.join(data))
 
 
+def decode_AISPec(data, name=None):
+    logger.debugfast("decode_AISpec")
+
+    antenna_count = ushort_unpack(data[:ushort_size])[0]
+
+    antenna_ids_length = ushort_size * antenna_count
+    antenna_ids = data[ushort_size:ushort_size + antenna_ids_length]
+
+    par = {
+        'AntennaCount': antenna_count,
+        'AntennaID': {
+            int.from_bytes(
+                antenna_ids[b+b:b+b+ushort_size],
+                'big'
+            ) for b in range(len(antenna_ids)//ushort_size)}
```

Comment:
"from_bytes" is python3 specific, and so far, we are still python2 retro compatible.
But good news, you can use "ushort_unpack" without prejudice as it looks like that it is still a lot faster than "from_bytes".
(Ex.: https://github.com/kyuupichan/electrumx/blob/master/docs/PERFORMANCE-NOTES)

I'm not sure, but I think that it could be better to use [...] as comprehension instead of {...}:
In the current case, the "set" will ensure that there is not duplicate, but on the other side, [...] would ensure to keep the order of what is reported by the reader, and if i'm not wrong, it might be legal to have multiple times the same antenna_id in the list.